src/commands/bootstrap.ts

```diff
@@ -78,24 +84,22 @@ module.exports = function bootstrap({ client, config }) {
         console.error('goodbye channel not found.');
         return;
       }
-      goodbyeChannel.send(`** ${member.user} has left us! :( **`);
+      if (goodbyeChannel.type !== 'text') {
+        console.error('log ');
+      }
+      (goodbyeChannel as TextChannel).send(
+        `** ${member.user} has left us! :( **`
+      );
     });
   }
 
-  for (const file of commands) {
-    const command = require(`${__dirname}/${file}`);
-    // Set a new command file in the Discord Collection
-    // With the key as the command prefix and the value as the exported command function
-    client.commands.set(command.prefix, command);
-  }
-
   //deleted message logging
   client.on('messageDelete', function (message) {
     //change channel name to match server configuration
-    const logChannel = message.guild.channels.cache.find(
+    const logChannel = message.guild?.channels.cache.find(
```

Comment:
Question mark here is breaking the deleted message logging function. 😶

Reply:
This feature was working on my end, let me debug to see if I can get it to break

Reply:
Adding the `LOG_MSG_CHANNEL` to the `.env` didn't allow the command to work until I removed the `?` and recompiled.

Reply:
I'm on node version (node -v) `12.17.0` and have `tsc` version (npx tsc -v) of `3.9.5`.

@nhcarrigan Do you see an error during build (its a TS error) or when that specific code is ran (its a JS error)?

Reply:
I recompiled with the `?` present and now it seems to be functioning fine. I must have made an error last time. 

Running the `npm run build` only throws the `cannot find type definition file for 'jest'` error. I'm running node version `14.2.0` and apparently only `3.9.3` of `tsc`.